wgsl/index.bs

```diff
@@ -5776,89 +5776,92 @@ The reason is that only functions without a return type can have such a [=behavi
     <td class="nowrap">{ }
     <td>
     <td class="nowrap">{Next}
-  <tr>
+  <tr algorithm="braced statement behaviour">
     <td class="nowrap">{|s|}
     <td>|s|: |B|
     <td class="nowrap">B
```

Comment:
I just noticed that this is `B` instead of `|B|`. I believe the latter would be better, and would allow removing B from the ignored vars list.